src/Spice86/ViewModels/CfgCpuViewModel.cs

```diff
@@ -33,10 +32,10 @@ public partial class CfgCpuViewModel : ViewModelBase {
 
     [ObservableProperty] private bool _isCfgCpuEnabled;
 
-    public CfgCpuViewModel(Configuration configuration, ExecutionContextManager executionContextManager, IPauseHandler pauseHandler) {
+    public CfgCpuViewModel(ExecutionContextManager executionContextManager, IPauseHandler pauseHandler) {
         _executionContextManager = executionContextManager;
         _performanceMeasurer = new PerformanceMeasurer();
-        IsCfgCpuEnabled = configuration.CfgCpu;
+        IsCfgCpuEnabled = true;
```

Comment:
I would not set this to enabled by default, but rather pass the Configuration object. It is the single source of truth to know if the CfgCpu is enabled.